src/sentry/static/sentry/app/plugins/components/issueActions.jsx

```diff
@@ -68,6 +80,86 @@ class IssueActions extends PluginComponentBase {
     );
   }
 
+  setDependentFieldState(fieldName, state) {
+    const dependentFieldState = {...this.state.dependentFieldState};
+    dependentFieldState[fieldName] = state;
+    this.setState({dependentFieldState});
+  }
+
+  loadOptionsForDependentField = async field => {
+    const formData = this.getFormData();
+
+    const url =
+      '/issues/' +
```

Comment:
Can we use string templates here, it might make this a bit easier to read

Reply:
@billyvg I just copy and pasted this but I agree, that's better